pkg/rpcservice/webhook_endpoints_list.go

```diff
@@ -0,0 +1,34 @@
+package rpcservice
+
+import (
+	"context"
+
+	"github.com/stripe/stripe-cli/pkg/requests"
+	"github.com/stripe/stripe-cli/pkg/stripe"
+	"github.com/stripe/stripe-cli/rpc"
+)
+
+// WebhookEndpointsList returns a list of webhook endpoints.
+func (srv *RPCService) WebhookEndpointsList(ctx context.Context, req *rpc.WebhookEndpointsListRequest) (*rpc.WebhookEndpointsListResponse, error) {
+	userConfig := srv.cfg.UserCfg
+	livemode := false
+
+	key, err := userConfig.Profile.GetAPIKey(livemode)
+	if err != nil {
+		return nil, err
+	}
+
+	endpoints := requests.WebhookEndpointsList(ctx, stripe.DefaultAPIBaseURL, "2019-03-14", key, &userConfig.Profile)
```

Comment:
Is it okay to pin this to a specific API version?

Reply:
the one other place using api version is also hardcoded. i made it into a constant to be shared between two places

Reply:
Ah okay, it's the specific version the CLI can read i am assuming?